ducktape/cluster/remoteaccount.py

```diff
@@ -605,12 +605,13 @@ class SSHOutputIter(object):
     """Helper class that wraps around an iterable object to provide has_next() in addition to next()
     """
 
-    def __init__(self, iter_obj, channel_file=None):
+    def __init__(self, iter_obj_f, channel_file=None):
         """
         :param iter_obj: An iterator
         :param channel_file: A paramiko ``ChannelFile`` object
         """
-        self.iter_obj = iter_obj
+        self.iter_obj_f = iter_obj_f
```

Comment:
docs right above this need updating. I'd also probably use `_func` instead to make it clear its a callable.